content/github/getting-started-with-github/getting-started-with-git/managing-remote-repositories.md

```diff
@@ -56,7 +56,7 @@ $ git remote add origin https://{% data variables.command_line.codeblock %}/octo
 To fix this, you can:
 * Use a different name for the new remote
 * Rename the existing remote repository
-* Delete the existing remote repository
+* Before adding the new remote, remove the existing one with `git remote remove origin`
```

Comment:
Thanks for suggesting this change ✨ 

Currently, this set of bullet points gives a list of ways to fix the problem without explaining how. Your change adds an example of the command for one of the points. I think the best fix is to link to the sections later in this document that explain how to rename and delete a remote. That way, users will have all the information and won't just follow the example command (which will only work if the remote that they want to delete is called `origin`).

```suggestion
* Delete the existing remote repository before you add the new remote. For more information, see "[Removing a remote repository](#removing-a-remote-repository)" below.
```